gslib/tests/test_context_config.py

```diff
@@ -403,3 +411,37 @@ def test_writes_and_deletes_certificate_file_storing_password_to_memory(
       # Test deletes certificate file.
       context_config._singleton_config._unprovision_client_cert()
       mock_remove.assert_called_once_with(test_config.client_cert_path)
+
+  @mock.patch(OPEN_TO_PATCH, new_callable=mock.mock_open)
+  @mock.patch.object(os, 'remove')
+  @mock.patch.object(subprocess, 'Popen')
+  def test_writes_and_deletes_unencrypted_certificate_file_without_storing_password(
```

Comment:
Is this test meant to check the gcloud case? Should we mention that in the testname or in the docstring?

Reply:
Added a comment!